modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/drag-and-drop/getDropHandler.es.js

```diff
@@ -18,9 +18,9 @@ import {
 	dropLayoutBuilderField,
 } from '../actions.es';
 import {
-	DRAG_DATA_DEFINITION_FIELD,
-	DRAG_FIELDSET,
-	DRAG_FIELD_TYPE,
+	DRAG_DATA_DEFINITION_FIELD_ADD,
+	DRAG_FIELDSET_ADD,
+	DRAG_FIELD_TYPE_ADD,
 } from './dragTypes.es';
 
 export const getDropHandler = ({dataDefinition, dataLayoutBuilder}) => {
```

Comment:
@diegonvs I think Aline and Milton probably forgot to remove this file. Since we have all the implementations in `useDrop` this file no longer makes sense and is no longer being used. Please can remove it in your PR. It is also necessary to remove this `useEffect` https://github.com/liferay-forms/liferay-portal/blob/bd1ec4db9005b677f34046bff05ad71e75b411c9/modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/data-layout-builder/DataLayoutBuilderContextProvider.es.js#L121-L128

cc @miltonmc @alinedoleron

Reply:
Removed! 

@interaminense @kevenleone